src/components/Header.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Header = () => {
+  return (
+ 		<header className="header-container">
+ 			<h1 className="header-text">New albums & singles</h1>
```

Comment:
This is okay and would certainly work, but it would be better if we made the `Header` component reusable. Instead of hardcoding the text here, we can update this component to accept some props that it will display.

Reply:
Also, based on the designs they provided have in the repo, `h1` might be too big. Perhaps we can update this to H3?

Reply:
Changes updated!